src/test/java/core/basesyntax/ValidationTest.java

```diff
@@ -0,0 +1,264 @@
+                "Test failed! The size isn't correct. Expected "
+                        + expectedStorageSize + " but was " + Storage.people.size() + '\n');
+
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: already existing login: "
+                        + validUserBob.getLogin() + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    public void register_NullLogin_NotOk() {
+        validUserBob.setLogin(null);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: login == null" + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void register_EmptyLogin_NotOk() {
+        validUserBob.setLogin("");
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: login is empty" + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void register_InvalidPasswordLength_NotOk() {
+        validUserBob.setPassword("12345");
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: not enough length for password: "
+                        + validUserBob.getPassword() + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void register_NullPassword_NotOk() {
+        validUserBob.setPassword(null);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: password == null"
+                        + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void register_NullAge_NotOk() {
+        validUserBob.setAge(null);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: age == null" + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void register_InvalidAge_NotOk() {
+        validUserBob.setAge(17);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: user's age < 18!"
+                        + '\n', EXCEPTION_MESSAGE));
+
+        validUserBob.setAge(-19);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(validUserBob),
+                String.format("%S should be thrown for: user's age < 18!"
+                        + '\n', EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void getNull_Ok() throws RegistrationException {
```

Comment:
What are you testing here? StorageDao? This is RegistrationServiceImplTest class. RegistrationServiceImpl should be tested only.
Same for other methods.